app/src/lib/stores/app-store.ts

```diff
@@ -1023,18 +1023,49 @@ export class AppStore {
     return null
   }
 
-  /** Update the menu with the names of the user's preferred apps. */
-  private updatePreferredAppMenuItemLabels() {
+  /**
+   * Update menu labels for editor, shell, and pull requests.
+   */
+  private async updateMenuItemLabels(repository?: Repository) {
     const editorLabel = this.selectedExternalEditor
       ? `Open in ${this.selectedExternalEditor}`
       : undefined
 
+    let prLabel: string | undefined
+
+    if (repository) {
+      prLabel = await this.getPullRequestLabel(repository)
```

Comment:
This doesn't need to be awaited anymore. I'm surprised TS doesn't complain about that 🤔

Reply:
Yes, that is weird indeed, I typically rely on `tsc` to tell me this kind of stuff... Do you think our linter may be suppressing some of this feedback?

Reply:
I doubt it. Linting and compilation are separate parts of the build process.